common/app/model/content.scala

```diff
@@ -197,15 +197,17 @@ final case class Content(
     } getOrElse (None,None)
 
     val articleMeta = if (tags.isUSMinuteSeries) {
-      Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries))
-    } else None
+      List(
+        Some("isMinuteArticle", JsBoolean(tags.isUSMinuteSeries)),
+        Some("enhanceMobileTweets", JsBoolean(true))
```

Comment:
exactly, so why can't the js just check `isMinuteArticle`? The human logic is "a US minute article should enhance tweets."